support/tech-resources/smtp-rest-api-performance.md

```diff
@@ -31,8 +31,14 @@ Follow these best practices in order to optimize SMTP injection rates (and ove
 * Keep the TCP connections open between messages.  Closing and opening connections on each message is much less efficient and can slow performance. Best practice is to reuse connections for no more than 10,000 messages and then create a new connection. This allows local and remote resources to be cleaned up efficiently.
 * Ensure your SMTP client has pipelining enabled, to reduce the roundtrip delay on some SMTP commands.
 
+Following these best practices for optimal use of SMTP injection:
+
+* On timeout or any 4XX error the best practice is to retry.
+* Use a timeout of at least 10 seconds for the SMTP authentication phase (AUTH command).
```

Comment:
I think 10 seconds is too short, given how long the RFC recommends for timeouts. I'd say at least 30 seconds is more realistic.